.github/workflows/ci.yml

```
@@ -2,88 +2,29 @@ name: Deploy Next.js site to Pages
+      id-token: write  # if your job requires the id-token permission
 
-      - name: Start Development Server
-        run: ${{ steps.detect-package-manager.outputs.runner }} run dev
-
-      - name: Upload Build Artifacts
-        uses: actions/upload-pages-artifact@v3
-        with:
-          path: .next
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)